osu.Game/Screens/Backgrounds/BackgroundScreenBeatmap.cs

```diff
@@ -2,18 +2,34 @@
 // See the LICENCE file in the repository root for full licence text.
 
 using osu.Framework.Allocation;
+using osu.Framework.Configuration;
 using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
 using osu.Framework.Graphics.Textures;
+using osu.Framework.Screens;
 using osu.Game.Beatmaps;
+using osu.Game.Configuration;
+using osu.Game.Graphics;
 using osu.Game.Graphics.Backgrounds;
 
 namespace osu.Game.Screens.Backgrounds
 {
     public class BackgroundScreenBeatmap : BlurrableBackgroundScreen
     {
         private WorkingBeatmap beatmap;
+        protected Bindable<double> DimLevel;
+        public Bindable<bool> UpdateDim;
 
-        public WorkingBeatmap Beatmap
+        protected float BackgroundOpacity => 1 - (float)DimLevel;
```

Comment:
just mode this inline. there's only one usage and following the logic right now feels convoluted.